mmdet/core/bbox/match_costs/match_cost.py

```diff
@@ -244,12 +244,17 @@ class DiceCost:
         pred_act (bool, optional): Whether to apply sigmoid to mask_pred.
             Defaults to False.
         eps (float, optional): default 1e-12.
+        naive_dice (bool, optional): If True, use the dice \
```

Comment:
If True, use the naive dice loss in which the power of the number in the denominator is the first power.
If Flase, use the second power that is adopted by K-Net and SOLO. Defaults to True.